src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala

```diff
@@ -85,9 +86,15 @@ class Submitter(stagingBaseDir: File,
     // EASY-1464 3.3.7 checksums
     _ <- samePayloadManifestEntries(stageBag, draftBag)
     // EASY-1464 step 3.3.9 Move copy to submit-to area
+    _ = stageBag.baseDir.parent.list.foreach(setRights(_))
     _ = stageBag.baseDir.parent.moveTo(submitDir)
   } yield ()
 
+  private def setRights(file: File) = {
+    file.addPermission(PosixFilePermission.GROUP_WRITE)
```

Comment:
When deployed this causes: `java.io.IOException: 'owner' parameter can't be a group`